#TEST CONFIGURATIONS:
n_query: 50
n_iter_per_query: 10

#PLANNING CONFIGURATIONS:
group_name: "cartesian_arm"
start_configuration: [0.2,0.2,0.2]
stop_configuration:  [2.8,2.8,2.8]
final_start_configuration: [0.2,2.8,2.8]
final_stop_configuration:  [2.8,0.2,0.2]
utopia_tolerance: 0.05
extend: false
max_distance: 0.25
checker_resolution: 0.01
parallel_checker_n_threads: 4

#REPLANNER CONFIGURATIONS:

# Available replanners:
# - MPRRT
# - DRRT*
# - DRRT
# - anytimeDRRT
# - MARS
# - MARSHA

replanner_type: "MARSHA"
dt_replan: 0.20
trj_execution_thread_frequency: 500
collision_checker_thread_frequency: 30
benchmark: false
scaling: 0.5
read_safe_scaling: false
joint_target_topic: "/joint_target"
unscaled_joint_target_topic: "/unscaled_joint_target"
overrides: ["/safe_ovr_1"]
virtual_obj:
  spawn_objs: true
  spawn_instants: [0.5, 0.7, 0.9, 1.1, 1.3, 1.5] #[0.5, 2, 3.5]
  obj_type: "sphere"
  obj_max_size: 0.1
  obj_vel: 0 #0.5
  dt_move: 0.1
  moves_before_change_direction: 10
  obs_pose_topic: "/poses"

MPRRT:
  n_threads_replan: 5

MARS:
  n_other_paths: 2
  reverse_start_nodes: true
  full_net_search: false
  dt_replan_relaxed: 0.20
  verbosity_level: 1
  display_other_paths: true

MARSHA:
  base_frame: "world"
  tool_frame: "end_effector"
  ssm_max_step_size: 0.025
  ssm_parallel: true
  ssm_threads: 4
  unaware_obstacles: ["manipulation/big_box/n_0","manipulation/box/n_0","manipulation/box/n_1"]
  poi_names: ["end_effector"]
  max_cart_acc: 2.5
  Tr: 0.15
  min_distance: 0.2
  v_h: 1.6

#VERBOSITY:
display: false
replanner_verbosity: false
display_timing_warning: false
display_replanning_success: true
display_replan_trj_point: false
display_replan_config: false
display_current_trj_point: false
display_current_config: false
